connectors/golden-tables/src/test/scala/io/delta/golden/GoldenTables.scala

```diff
@@ -378,6 +378,7 @@ class GoldenTables extends QueryTest with SharedSparkSession {
 
     val commitInfoFile = CommitInfo(
       version = Some(0L),
+      inCommitTimestamp = None,
```

Comment:
This is broken on current master.

Reply:
what does this mean?

Reply:
there was a commit in `delta-spark` that updated the argument list. Our current CI setup just runs the `delta-spark` tests and not its dependencies. We need to revisit our CI trigger job to make sure all run tests for all dependent modules. 

I can make a separate PR for this change if you would like.